src/core/runtime.d

```diff
@@ -21,6 +21,8 @@ else version (WatchOS)
 
 /// C interface for Runtime.loadLibrary
 extern (C) void* rt_loadLibrary(const char* name);
+/// ditto
+version (Windows) extern (C) void* rt_loadLibrary(const WCHAR* name);
```

Comment:
Duplicate declarations for `version (Windows)` like this are rejected by LDC, and rightly so IMO.

I don't think it's a good idea to have a C interface which diverges for Windows. The different param type isn't part of the mangled name, so you won't get link errors, but ugly crashes for external (C[++]...) code using that interface.

Reply:
@edi33416 per https://github.com/dlang/druntime/pull/2392#pullrequestreview-186042151 that should be added back as `rt_loadLibraryW` that uses `WCHAR*`

Reply:
@thewilsonator I had the same fix in mind, but please see @WalterBright [comments](https://issues.dlang.org/show_bug.cgi?id=19498#c1) on the issue.

Reply:
I can't move the `core.runtime.loadLibrary()` implementation to `rt.dmain2.rt_loadLibrary()`, as `rt_loadLibrary()` takes a `const char*` instead of `const char[]`, so it's losing the length information (unless it's safe to call `strlen(name)` )